tests/integration/conformance/sanity_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
```

Comment:
what is this test for?

Reply:
This is a way to ensure that all known resources are accounted for, for testing purposes. If somebody introduces a new resource type without adding a new conformance tests, then this test will fail.